src/components/headMenu/headMenu.styles.css

```diff
@@ -0,0 +1,23 @@
+.head-menu {
+  background: #efefef;
+  max-height: 60px;
+  min-width: 1200px;
```

Comment:
DO you mean max-width? Because on shrinking the viewport, your content is overflowing.

Reply:
Yes I have not used media query as of now means this is not responsive as of now Its for desktop only as of now

Reply:
max-width is not just used in media queries but it is also a standalone CSS property. Clearly you do not understand the use case.

Reply:
Removed max-width no need to set it